std_msgs/Header    header
geometry_msgs/Vector3 position
geometry_msgs/Vector3 orientation

float64 wbad
float64 wbas
float64 speed
float64 gain
int16 closed

int16 trial
int16 servoAngle
int16 valve1
int16 valve2
int16 valve3
int16 quadrant

float64 DCoffset

float64 impose
float64 imposeResponse

float64 imposeResponseSmooth
float64 imposeHeading
float64 imposeResponseHeading
float64 compensation

geometry_msgs/Vector3 object1Pos
geometry_msgs/Vector3 object2Pos

int16 reset

string key

